libs/libc/wqueue/work_cancel.c

```diff
@@ -44,12 +44,12 @@
  *
  * Description:
  *   Cancel previously queued work.  This removes work from the work queue.
- *   After work has been cancelled, it may be re-queue by calling
+ *   After work has been cancelled, it may be re-queued by calling
```

Comment:
```suggestion
 *   After work has been cancelled, it may be requeued by calling
```
nit: This is both written as "re-queue" and "requeue". For consistency, we need to decide for a single one. According to Google, the one that has the most references is "requeue". So I'd go with that.

Reply:
Good catch! Fixed, see latest force push